src/styles/components/components/SSIEntityIcon/index.ts

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components/native'
+
+export const SSIEntityIconStyled = styled.View`
+ margin-top: 5px;
```

Comment:
i would not expect this margin to be present as this will mover the circle 5 pixels downwards. any reason why we do this?

Reply:
Removed the margin